tests/External/.gitkeep

```diff
@@ -0,0 +1 @@
+Once you add an external test, you can delete this file.
```

Comment:
https://www.aaronsaray.com/2021/better-alternative-to-gitkeep

Reply:
Deleted `.gitkeep` and added a README.